app/src/main/java/com/arygm/quickfix/model/account/Account.kt

```diff
@@ -8,5 +8,6 @@ data class Account(
     val lastName: String,
     val email: String,
     val birthDate: Timestamp,
-    val isWorker: Boolean = false
+    val isWorker: Boolean = false,
+    val activeChats: List<String> = emptyList(),
```

Comment:
As we said in the meeting, i think we should have separate messages as a user and as a worker, not unified for the account, but that's just my opinion.

Reply:
I will definitely keep that in mind for our next meeting but we can leave that aside for now as we need to fast track the feature